service/history/hsm/tree.go

```diff
@@ -170,6 +196,7 @@ func (n *Node) Dirty() bool {
 type TransitionOutputWithCount struct {
 	TransitionOutput
 	TransitionCount int64
+	NodePath        []Key
 }
 type PathAndOutputs struct {
```

Comment:
You can remove this struct now that we have the OpLog.

Reply:
Done